spec/models/event_spec.rb

```diff
@@ -23,62 +23,21 @@
 
 describe 'Event model', type: :model do
 
-  let(:event) { FactoryBot.build(:league) }
+  let(:event) { FactoryBot.build(:event) }
 
   it 'should not validate without name' do
-    league = FactoryBot.build(:league, name: nil)
-    expect(league.valid?).to eq(false)
+    event = FactoryBot.build(:event, name: nil)
+    expect(event.valid?).to eq(false)
```

Comment:
Why is this test not in the shared example?

Reply:
because it is validated in the events model and not in the league or tournament model.
`validates :name, :discipline, :game_mode, presence: true ` is the first line in the Event class. All these validations are tested in Event_spec.rb